app/templates/components/widgets/forms/rich-text-editor.hbs

```diff
@@ -13,5 +13,15 @@
     <a href="#" class="ui button" data-content="{{t 'Redo (Ctrl+Y)'}}" data-wysihtml5-command="redo"><i class="repeat icon"></i></a>
     <a href="#" class="ui button" data-content="{{t 'Line Break'}}" data-wysihtml5-command="insertLineBreak"><i class="level down icon"></i></a>
   </div>
+  <div class="ui icon buttons">
+    <a href="#" class="ui button" data-wysihtml5-command="createLink" data-content="{{t 'Insert Link'}}" ><i class="linkify icon"></i></a>
+  </div>
+    <div data-wysihtml5-dialog="createLink" class="mb-8">
```

Comment:
Can you check if this works?

```suggestion
    <div data-wysihtml5-dialog="createLink" class="mb-8" style="display: 'none' ">
```

Refers https://github.com/Voog/wysihtml/blob/3f7a34a2721813c81c3b2205a9bfff8b8e472eff/examples/advanced_div.html#L262-L268

Reply:
thanks man :+1: